core/tests/webdriverio_utils/LearnerDashboardPage.js

```diff
@@ -0,0 +1,265 @@
+  };
+
+  this.expectTitleOfExplorationSummaryTileToBeHidden = async function(title) {
+    var items = await $$(
+      `.e2e-test-exp-summary-tile-title=${title}`);
+    expect(items.length).toBe(0);
+  };
+
+  this.expectTitleOfExplorationSummaryTileToMatch = async function(title) {
+    // This explorationTitleArray is an Array but it will have only one element
+    // that is the exploration with the title passed as a parameter.
+    var explorationTitle = $(
+      `.e2e-test-exp-summary-tile-title=${title}`);
+    var titleOfExplorationSummary = await action.getText(
+      'Exploration title', explorationTitle);
+    expect(titleOfExplorationSummary).toMatch(title);
+  };
+
+  this.expectNameOfTopicInEditGoalsToMatch = async function(name) {
+    await waitFor.visibilityOf(
+      editGoalsTopicName,
+      'Topic in Edit Goals takes too long to appear');
+    await waitFor.textToBePresentInElement(
+      editGoalsTopicName, name,
+      `Text "${name}" taking too long to be present in editGoalsTopic`);
+    var topicName = $(
+      `.e2e-test-topic-name-in-edit-goals=${name}`);
+    expect(await action.getText('Topic Name', topicName)).toMatch(name);
+  };
+
+  this.expectNameOfTopicInCurrentGoalsToMatch = async function(name) {
+    await waitFor.visibilityOf(
+      currentGoalsTopicName,
+      'Topic in Current Goals takes too long to appear');
+    await waitFor.textToBePresentInElement(
+      currentGoalsTopicName, name,
+      `Text "${name}" taking too long to be present in currentGoalsTopic`);
+    var topicName = $(
+      `.e2e-test-topic-name-in-current-goals=${name}`);
+    expect(await action.getText('Topic Name', topicName)).toMatch(name);
+  };
+
+  this.expectNameOfTopicInSkillProficiencyToMatch = async function(name) {
+    await waitFor.visibilityOf(
+      skillProficiencyTopicTitle,
+      'Topic in Skill Proficiency takes too long to appear'
+    );
+    await waitFor.textToBePresentInElement(
+      skillProficiencyTopicTitle, name,
+      `Text "${name}" taking too long to be present in skillProficiencyTopic`);
+    var topicName = $(
+      `.e2e-test-skill-proficiency-topic-title=${name}`);
+    expect(await action.getText('Topic Name', topicName)).toMatch(name);
+  };
+
+  this.expectNameOfTopicInCompletedGoalsToMatch = async function(name) {
+    await waitFor.visibilityOf(
+      completedGoalsTopicName,
+      'Topic in completed goals takes too long to appear'
+    );
+    await waitFor.textToBePresentInElement(
+      completedGoalsTopicName, name,
+      `Text "${name}" taking too long to be present in completedGoalsTopic`);
+    var topicName = $(
+      `.e2e-test-completed-goals-topic-name*=${name}`);
+    expect(await action.getText('Topic Name', topicName)).toMatch(name);
+  };
+
+  this.expectNumberOfTopicsInSuggestedForYou = async function(value) {
+    var topicNamesInLearnerTopicSummaryTiles = await $$(
+      '.e2e-test-learner-topic-summary-tile-title');
```

Comment:
Was this not used at the top? Ditto Elsewhere.

Reply:
Done.